daemon/api_snaps_test.go

```diff
@@ -1410,10 +1410,14 @@ func (s *snapsSuite) TestSnapManyInfosReturnsRefreshInhibitProceedTime(c *check.
 		switch snap["name"] {
 		case "snap-a":
 			refreshInhibit := snap["refresh-inhibit"].(map[string]interface{})
-			c.Assert(refreshInhibit["proceed-time"], check.Equals, expectedProceedTimeA.Format(time.RFC3339Nano), testCmt)
+			proceedTime, err := time.Parse(time.RFC3339Nano, refreshInhibit["proceed-time"].(string))
```

Comment:
alternative would be to consistently return .UTC() through the API, but we're already returning local time values in some endpoints, so I guess it is what it is

Reply:
Yes, I thought of doing this, but stopped because it wouldn't be consistent with other endpoints.